### ISSUE TITLE
About skipping label 0 when constructing dataset 

### ISSUE THREAD

**Issue by XBR-1111** [hennande/Adv-ALSTM#6]
Firstly, thank you for sharing your code and data. 
However, when loading data, the code just skips samples with label 0, whose next-day return is between -0.5% and 0.55%. It's not good especially for test set. For unknown data in test set, It actually have 3 labels, and the code destroys the distribution of test data.
I just wonder whether my understand is correct.

**Comment by fulifeng** [comment]
Yes, this is not a good way to construct label. But this is a hard choice. During my rejected submission, there are comments like "Should follow the same setting as previous work (StockNet)".
